pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -919,11 +919,11 @@ func (m *kubeGenericRuntimeManager) doBackOff(pod *v1.Pod, container *v1.Contain
 	}
 
 	klog.V(3).InfoS("Checking backoff for container in pod", "containerName", container.Name, "pod", klog.KObj(pod))
-	// Use the finished time of the latest exited container as the start point to calculate whether to do back-off.
-	ts := cStatus.FinishedAt
+
 	// backOff requires a unique key to identify the container.
 	key := getStableKey(pod, container)
-	if backOff.IsInBackOffSince(key, ts) {
+	now := time.Now()
+	if backOff.IsInBackOffSinceUpdate(key, now) {
```

Comment:
ok, changing to `IsInBackOffSinceUpdate` makes it working

Reply:
I do not understand the reasoning for no longer looking at the finished timestamp of the last exited container versus the present timestamp, how did that change fix the core issue?  can you elaborate?